packages/react/src/components/molecules/HamburgerNav/index.js

```diff
@@ -503,44 +507,44 @@ export const HamburgerNavItem = ({
   }, [itemRef, buttonRef, contentRef, ulRef]);
   return(
     <li ref={itemRef} role="none" className={classes} tabIndex="-1">
-      {isCovid ? (
+      {hasSubNav ? (
+        <>
+          <button ref={buttonRef} type="button" role="menuitem" id={`button${index}`} className="ma__main__hamburger-nav__top-link js-main-nav-hamburger__top-link" aria-haspopup="true" tabIndex="0">
```

Comment:
The use of `menuitem` here (and below) is actually a deceptive accessibility issue.

The `button`s and the nested `<a>` tags should not have the `menuitem` role. [`menuitem`s](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role) should only be the children of elements with the [`menu` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role), and the `menu`/`menuitem` groupings are not used to indicate any navigation, but rather a menu of actions similar to a desktop application, like the menu when right clicking an item.

For web navigation, it should be marked up as a `<nav>` and a nested `<ul>`, with `<li>` inside for each item, and a single `<a>` inside each `<li>`. For sub navs, a new `<ul>` is nested inside the given `<li>` instead of a `<a>`. The natural semantics of those elements provides the right accessible markup, without the need for any added `role`s.

Reply:
@bronzehedwick  regarding the roles, I actually didn't make any changes in this PR. This one is only aiming to fix the nav link issue. I have another open PR where I edited the roles - but I got some feedback from @ygannett on the changes. I will review her feedback in detail tomorrow. I will discuss your comments with her in that context. Thanks for your thorough review!

Reply:
Forgot to link to the other PR --> https://github.com/massgov/mayflower/pull/1792